agent/ecscni/netconfig_windows.go

```diff
@@ -25,12 +25,10 @@ import (
 
 // NewVPCENIPluginConfigForTaskNSSetup is used to create the configuration of vpc-eni plugin for task namespace setup.
 func NewVPCENIPluginConfigForTaskNSSetup(eni *eni.ENI, cfg *Config) (*libcni.NetworkConfig, error) {
+	// Use the DNS server addresses of the instance ENI it would belong in the same VPC as
+	// the task ENI and therefore, have same DNS configuration.
```

Comment:
what happen when the dhcp options of the vpc changes between agent start and task start? can we end up having different dns servers between `cfg. InstanceENIDNSServerList` and `eni.DomainNameServers ` in that case?

Reply:
DHCP would be enabled for the task namespace, so the appropriate DNS configuration would be updated automatically.